tests/functional/spec/features/calculated_summary/calculated_summary_test_case.js

```diff
@@ -422,6 +422,51 @@ class TestCase {
       );
     });
   }
+
+  testNegative(schema, firstAnswerValue, secondAnswerValue, thirdAnswerValue, fourthAnswerValue, expectedTotalValue, expectedAnswerValue) {
```

Comment:
Paired with my comment above, I think you can remove the `expectedAnswerValue` as a param

Reply:
Happy this is resolved. It has not been removed but is now an array. It could be dynamically calculated but left as is since it is minor.